NetworkChat/Client.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Sockets;
+using System.IO;
+
+
+namespace NetworkChat
+{
+    class Client
+    {
+        TcpClient client { get; set; }
+        string Name { get; set; }
+        string Login { get; set; }
+
+        private readonly int Port;
+        private readonly IPAddress Ip;
+        private TcpClient My_client;
+
+        public Client(int port, IPAddress ip, string name)
+        {
+            Port = port;
+            Ip = ip;
+            My_client = new TcpClient();
+            Name = name;
+        }
+
+
+        public async Task Start()
+        {
+            await client.ConnectAsync(Ip, Port);
+            Console.WriteLine("Client Start");
+
+            string cur_string = "";
+
+            while (cur_string != "exit")
+            {
+                cur_string = await Read();
+                if (cur_string == "exit") break;
+                cur_string = await Write();
+            }
+
+            client.Close();
+        }
+
+
+        public async Task<string> Read()
+        {
+            StreamReader sr = new StreamReader(client.GetStream());
```

Comment:
```suggestion
            var sr = new StreamReader(client.GetStream());
```
В современном C# никогда не надо писать тип дважды

Reply:
А правильнее using var. Почитайте про IDisposable и зачем оно нужно